[2024-12-02T11:19:15.309+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-02T11:19:15.345+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.save_data manual__2024-12-02T11:19:02.597239+00:00 [queued]>
[2024-12-02T11:19:15.360+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.save_data manual__2024-12-02T11:19:02.597239+00:00 [queued]>
[2024-12-02T11:19:15.360+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-12-02T11:19:15.381+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): save_data> on 2024-12-02 11:19:02.597239+00:00
[2024-12-02T11:19:15.395+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=392) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-02T11:19:15.396+0000] {standard_task_runner.py:72} INFO - Started process 398 to run task
[2024-12-02T11:19:15.397+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'save_data', 'manual__2024-12-02T11:19:02.597239+00:00', '--job-id', '2029', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmp4tzl2qvs']
[2024-12-02T11:19:15.398+0000] {standard_task_runner.py:105} INFO - Job 2029: Subtask save_data
[2024-12-02T11:19:15.472+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.save_data manual__2024-12-02T11:19:02.597239+00:00 [running]> on host 59d51ca962df
[2024-12-02T11:19:15.666+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T11:19:02.597239+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T11:19:02.597239+00:00'
[2024-12-02T11:19:15.667+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-02T11:19:15.667+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-02T11:19:15.668+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2024-12-02 11:19:15.346273+00:00
[2024-12-02T11:19:15.668+0000] {logging_mixin.py:190} INFO - Dag name:get_inaturalist_data_atropelamento and current dag run status:running
[2024-12-02T11:19:15.668+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-02T11:19:15.669+0000] {inaturalist_dag.py:197} INFO - No new observations to store.
[2024-12-02T11:19:15.669+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-02T11:19:15.686+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-02T11:19:15.686+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=save_data, run_id=manual__2024-12-02T11:19:02.597239+00:00, execution_date=20241202T111902, start_date=20241202T111915, end_date=20241202T111915
[2024-12-02T11:19:15.702+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-02T11:19:15.702+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-02T11:19:15.703+0000] {logging_mixin.py:190} INFO - Dag name:get_inaturalist_data_atropelamento queued_at:2024-12-02 11:19:02.634318+00:00
[2024-12-02T11:19:15.703+0000] {logging_mixin.py:190} INFO - Task hostname:59d51ca962df operator:_PythonDecoratedOperator
[2024-12-02T11:19:15.731+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-02T11:19:15.747+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
